package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "@meshok/eslint-config",
```

Comment:
@codewylle man, you have 4-size tab here and 2-size space in other json files

Reply:
Your meshok is strong, man!